nerfstudio/exporter/exporter_utils.py

```diff
@@ -304,7 +308,15 @@ def collect_camera_poses_for_dataset(dataset: Optional[InputDataset]) -> List[Di
     # new cameras are in cameras, whereas image paths are stored in a private member of the dataset
     for idx in range(len(cameras)):
         image_filename = image_filenames[idx]
-        transform = cameras.camera_to_worlds[idx].tolist()
+        if camera_optimizer is None:
+            transform = cameras.camera_to_worlds[idx].tolist()
+        else:
+            # print('exporting optimized camera pose for camera %d' % idx)
+            camera = cameras[idx]
```

Comment:
If I set `camera = cameras[idx:idx+1]` instead, would it simplify how I would handle input in `apply_to_camera()`? I don't really like the fact that `apply_to_camera()` can handle different input shapes as it is implemented now.